app/renderer/components/preferences/payment/addFundsDialog/addFundsDialog.js

```diff
@@ -0,0 +1,372 @@
+        <div>
+          <AboutPageSectionTitle canWrap data-l10n-id='batContributionTitle' />
+          <p data-l10n-id='batContributionText1'
+            className={css(styles.addFundsDialog__main__text)}
+          />
+          <p data-l10n-id='batContributionText2'
+            className={css(styles.addFundsDialog__main__text)}
+          />
+        </div>
+        <p data-l10n-id='batContributionText3'
+          className={css(
+            styles.addFundsDialog__main__text,
+            styles.addFundsDialog__main__text_small
+          )}
+        />
+      </div>
+    )
+  }
+
+  get addFundsWizardMain () {
+    return (
+      <div data-test-id='addFundsWizardMain'
+        className={css(styles.addFundsDialog__inherit)}
+      >
+        <div>
+          <header data-l10n-id='addFundsWizardMainHeader' />
+          <p data-l10n-id='addFundsWizardMainOptions'
+            className={css(
+              styles.addFundsDialog__main__text,
+              styles.addFundsDialog__main__text_bold
+            )}
+          />
+          <BrowserButton
+            groupedItem
+            secondaryColor
+            custom={[
+              styles.currencyIcon,
+              styles.ethIcon
+            ]}
+            onClick={this.something}
+          />
+          <BrowserButton
+            groupedItem
+            secondaryColor
+            custom={[
+              styles.currencyIcon,
+              styles.btcIcon
+            ]}
+            onClick={this.something}
+          />
+          <BrowserButton
+            groupedItem
+            secondaryColor
+            custom={[
+              styles.currencyIcon,
+              styles.ltcIcon
+            ]}
+            onClick={this.something}
+          />
+          <BrowserButton
+            groupedItem
+            secondaryColor
+            custom={[
+              styles.currencyIcon,
+              styles.batIcon
+            ]}
+            onClick={this.something}
+          />
+        </div>
+        <p>
+          <span data-l10n-id='addFundsWizardMainReminder'
+            className={css(styles.addFundsDialog__main__text_small)}
+          />&nbsp;
+          <a data-l10n-id='theFAQ'
+            className={css(styles.addFundsDialog__main__text_small)}
+            href='https://brave.com/faq-payments/#brave-payments'
+            target='_blank'
+            rel='noopener'
+          />
+        </p>
+      </div>
+    )
+  }
+
+  onCopy () {
+    // TODO
+    // if (!this.addFundsInputNode) {
+    //   return
+    // }
+  }
+
+  get copyToClipboardButton () {
+    return (
+      <ClipboardButton
+        className={globalStyles.appIcons.clipboard}
+        copyAction={this.onCopy}
+      />
+    )
+  }
+
+  get addFundsWizardAddress () {
+    return (
+      <div>
+        <header data-l10n-id='addFundsWizardAddressHeader' />
+        <div className={css(styles.addFundsWizardAddress)}>
+          <main className={css(styles.addFundsWizardAddress__address)}>
```

Comment:
I'd pick `styles.wizard` and `styles.wizard__address` to shrink the length (because it should be already clear what kind of wizard it is, from the name `addFundsWizardAddress`).